readme.md

```diff
@@ -0,0 +1,191 @@
+# remark for Visual Studio Code
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+Visual Studio Code extension to format and lint markdown files with remark.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Use](#use)
+    *   [Formatting](#formatting)
+*   [Plugins](#plugins)
+*   [Syntax](#syntax)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This project is a Visual Studio Code (VS Code) extension that you can use in
+your editor to inspect and change markdown files.
+This extension is built around remark, which is a very popular ecosystem of
+plugins that work with markdown.
+You can choose from the 150+ plugins that already exist or make your own.
+
+See [remark][] for info on what the remark ecosystem is.
+
+## When should I use this?
+
+You can use this extension if you want to check (lint) and format markdown
+files from within your editor.
+
+To configure this extension, you define your preferred markdown style in a
+configuration file (`.remarkrc`, `.remarkrc.js`, etc. or in `package.json`).
+This file is picked up by `vscode-remark` and other tools (useful for
+contributors that don’t use VS Code).
+
+The configuration file is also used by [`remark-cli`][remark-cli], which is
+recommended to be used alongside `vscode-remark`, as an npm script and/or in
+CI, to enforce the markdown style.
+
+## Use
+
+To start using this for a project, [`remark-cli`][remark-cli] needs to be
+configured first.
+First, install `remark-cli` and a plugin or preset.
+Let’s use `remark-preset-lint-recommended` for our example.
+
+```sh
+npm install remark-cli remark-preset-lint-recommended
+```
+
+Now create a file named `.remarkrc.json` with the following content:
+
+```json
+{
+  "plugins": ["preset-lint-recommended"]
+}
+```
+
+Now let’s create a markdown file `readme.md` and add the following content:
+
+```markdown
+# My project
+
+This is a [broken link][]
+
+- A `*` should be used for list items
+- Also 4 spaces should be used for indentation
+
+[unused reference]: https://example.com
+```
+
+Now any syntax which doesn’t comply with `remark-preset-lint-recommended` will
+be reported using squiggly lines in your editor and in the `Problems` pane.
+
+### Formatting
```

Comment:
Not sure this should be an H3 under `## Use` 🤔. Maybe an H2?

Reply:
It should definitely under `Use`, as it’s a way to use the extension. I added a H3 `Linting`, because the section on top of this explains the linting part.

Reply:
`Use` should be a short but useful explanation of how to minimally use a thing.
It’s not a place to list every possible way something could be used.

You can instead use `API`, `CLI`, etc. if you want to expand on the interface in detail.
If this is about examples, use an `## Examples`, with another `### Example: formatting ...`

Reply:
IMHO it feels like you’re trying to force this into a format that doesn’t really work. Both **linting** and **formatting** are **use** cases of this extension.

Anyway, I don’t feel like this is worth the discussion, so I removed the `Linting` header and made the `Formatting` header level 2.